addons/strings/fnc_find.sqf

```diff
@@ -35,31 +35,26 @@ SCRIPT(find);
 
 // ----------------------------------------------------------------------------
 
-params ["_haystack","_needle", ["_initialIndex",0]];
+params ["_haystack", "_needle", ["_initialIndex", 0]];
 
 private _start = -1;
-private _ret = -1;
+private _return = -1;
 
-if (typeName _haystack != "STRING") exitWith {
-    -1
-};
-if (typeName _needle != "STRING") exitWith {
-    -1
-};
+if (typeName _haystack != "STRING") exitWith {-1};
+if (typeName _needle != "STRING") exitWith {-1};
```

Comment:
Should change it to `_var isEqualType ""` too.

Reply:
Done. Also, changed the private's and exitWith's positions. There's no point in assigning vars if the script might be terminated in the next line.